.travis/test.sh

```diff
@@ -4,7 +4,7 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
   source $HOME/miniconda/bin/activate
   export PATH="$HOME/miniconda/bin:$PATH"
   conda activate abc-dev
-  #python -m pytest -v
+  python -m pytest -v
```

Comment:
this has been commented out so reactivating it for now so the docs build on mac osx too.

but the build seems to be overall a bit messy... we can revisit in the future.